### ISSUE TITLE
Slider 'change' event is fired twice

### ISSUE THREAD

**Issue by zevisert** [material-components/material-components-web#5789]
**Describe the bug**
`mwc-slider`'s `'change'` event is fired twice on changes to the slider's value.

**To Reproduce**
1. Go to the official slider demo https://mwc-demos.glitch.me/demos/slider.html
2. Open Devtools Console (Ctrl+Shift+J / Cmd+Opt+J)
3. Attach an event listener to the sliders:
   *Copy this into the console*
  ```js
  document.querySelectorAll('mwc-slider').forEach(slider => slider.addEventListener('change', console.log))
  ```
4. Adjust a slider and observe that two `CustomEvents` are logged

**Expected behavior**
A slider's `'change'` event should only be fired once, as per the [docs](https://github.com/material-components/material-components-web-components/tree/master/packages/slider#events):
> Change: Fired when the value changes and the user has finished interacting with the slider.

**Screenshots**
![image](https://user-images.githubusercontent.com/11222441/74996344-26769a80-5408-11ea-9d65-b4b6e561f49f.png)

**Browser Version (please complete the following information):**
 - OS: Windows
 - Browser: Chromium Edge
 - Version: 80.0.361.56 (Official build) (64-bit)

**Additional context**
- Perhaps the docs mean to say that the change event fires when the value has changed **or** the user is done interacting? I wouldn't think so, however since the `'input'` event exists for the former.

**Comment by e111077** [comment]
issue moved

**Comment by e111077** [comment]
fixed by #5845. May take up to an hour to propagate to MWC canary
